v2.1/export.md

```diff
@@ -0,0 +1,160 @@
+
+## Cancelling Export Jobs
+
+After the export has been initiated, you can cancel it with [`CANCEL JOB`](cancel-job.html).
+
+## Synopsis
+
+{% include sql/{{ page.version.version }}/diagrams/export.html %}
+
+{{site.data.alerts.callout_info}}The <code>EXPORT</code> statement cannot be used within a <a href=transactions.html>transaction</a>.{{site.data.alerts.end}}
+
+## Required Privileges
+
+Only the `root` user can run [`EXPORT`](export.html).
+
+## Parameters
+
+| Parameter | Description |
+|-----------|-------------|
+| `file_location` | Specify the URL of the file location where you want to store the exported CSV data.|
+| `WITH kv_option` | Control your import's behavior with [these options](#export-options). |
+| `select_stmt` | Specify the query whose result you want to export to CSV format. |
+| `table_name` | Specify the name of the table you want to export to CSV format. |
+
+### Export File URLs
+
+URLs for the files you want to export to must use the following format:
+
+{% include external-urls-v2.0.md %}
+
+### Export Options
+
+You can control the [`EXPORT`](export.html) process's behavior using any of the following key-value pairs as a `kv_option`.
+
+#### `delimiter`
+
+If not using comma as your column delimiter, you can specify another Unicode character as the delimiter.
+
+<table>
+	<tbody>
+		<tr>
+			<td><strong>Required?</strong></td>
+			<td>No</td>
+		</tr>
+		<tr>
+			<td><strong>Key</strong></td>
+			<td><code>delimiter</code></td>
+		</tr>
+		<tr>
+			<td><strong>Value</strong></td>
+			<td>The unicode character that delimits columns in your rows</td>
+		</tr>
+		<tr>
+			<td><strong>Example</strong></td>
+			<td>To use tab-delimited values: <code>WITH temp = '...', delimiter = e'\t'</code></td>
+		</tr>
+	</tbody>
+</table>
+
+#### `comment`
+
+Do not export rows that begin with this character.
+
+<table>
+	<tbody>
+		<tr>
+			<td><strong>Required?</strong></td>
+			<td>No</td>
+		</tr>
+		<tr>
+			<td><strong>Key</strong></td>
+			<td><code>comment</code></td>
+		</tr>
+		<tr>
+			<td><strong>Value</strong></td>
+			<td>The unicode character that identifies rows to skip</td>
+		</tr>
+		<tr>
+			<td><strong>Example</strong></td>
+			<td><code>WITH temp = '...', comment = '#'</code></td>
+		</tr>
+	</tbody>
+</table>
+
+#### `nullas`
+
+Convert values to SQL *NULL* if they match the specified string.
+
+<table>
+	<tbody>
+		<tr>
+			<td><strong>Required?</strong></td>
+			<td>No</td>
+		</tr>
+		<tr>
+			<td><strong>Key</strong></td>
+			<td><code>nullas</code></td>
+		</tr>
+		<tr>
+			<td><strong>Value</strong></td>
+			<td>The string that should be converted to <em>NULL</em></td>
+		</tr>
+		<tr>
+			<td><strong>Example</strong></td>
+			<td>To use empty columns as <em>NULL</em>: <code>WITH temp = '...', nullas = ''</code></td>
+		</tr>
+	</tbody>
+</table>
+
+## Examples
+
+### Export Entire Table
+
+~~~ sql
```

Comment:
Let's add the copy-to-clipboard feature.

Reply:
Done.